### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by safijalalzai** [HappyFamilyCS/DNS-server-Python#9] [General]
Please add information about yourselves into readme file and introduction file

**Comment by KhalidMubaraze** [comment]
Safiullah
What did you do?

**Comment by KhalidMubaraze** [comment]
please do your work .

**Reply by safijalalzai** [reply]
I completed my works .
now you can do exceptions handling in code.

**Comment by KhalidMubaraze** [comment]
Thanks 
I added my info to introduction file

**Reply by safijalalzai** [reply]
Thanks Mr Mubariaze

**Comment by EsmatYasir** [comment]
Ok i will try its.

**Comment by abdulsaboorhemat1500** [comment]
hello happy family members
